tools/clang/test/CodeGenSPIRV_Lit/intrinsics.dot4add.hlsl

```diff
@@ -1,23 +1,28 @@
-// RUN: not %dxc -E frag -T ps_6_4 -fspv-target-env=vulkan1.1 -fcgl  %s -spirv  2>&1 | FileCheck %s
+// RUN: %dxc -E main -T ps_6_4 -fspv-target-env=vulkan1.1 -fcgl  %s -spirv  2>&1 | FileCheck %s
 
-uint frag(float4 vertex
-          : SV_POSITION) : SV_Target {
-  uint acc = 0;
-
-  // CHECK: 8:14: error: dot4add_u8packed intrinsic function unimplemented
-  acc += 1 + dot4add_u8packed(vertex.x, vertex.y, uint(vertex.y));
+float2 f2;
 
-  // CHECK: 11:14: error: dot4add_i8packed intrinsic function unimplemented
-  acc += 2 + dot4add_i8packed(vertex.z, vertex.w, int(vertex.x));
-
-  // CHECK: 14:10: error: dot4add_u8packed intrinsic function unimplemented
-  acc += dot4add_u8packed(vertex.x, vertex.y, uint(vertex.y)) + 1;
+float2 main(uint4 inputs : Inputs0, uint acc0 : Acc0, int acc1 : Acc1) : SV_Target {
+// CHECK-LABEL: %bb_entry = OpLabel
```

Comment:
Was this line left in unintentionally?

Reply:
It was copied from another test, I wasn't sure if it's needed or not. Removed.